x-pack/plugins/cases/docs/openapi/paths/api@cases.yaml

```diff
@@ -0,0 +1,152 @@
+post:
+  description: >
+    Creates a case.
+    You must have all privileges for the **Cases** feature in the **Management**, **Observability**, or **Security** section of the Kibana feature privileges, depending on the owner of the case you're creating.
+  parameters:
+    - $ref: ../components/headers/kbn_xsrf.yaml
+  requestBody:
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            connector:
+              description: An object that contains the connector configuration.
+              type: object
+              properties:
+                $ref: '../components/schemas/connector_properties.yaml'
+            description:
+              description: The description for the case.
+              type: string
+            owner:
+              $ref: '../components/schemas/owners.yaml'
+            settings:
+              description: An object that contains the case settings.
+              type: object
+              properties:
+                syncAlerts:
+                  description: Turns alert syncing on or off.
+                  type: boolean
+            tags:
+              description: The words and phrases that help categorize cases. It can be an empty array.
+              type: array
+              items:
+                type: string
+            title:
+              description: A title for the case.
+              type: string
+          required:
+            - connector
+            - description
+            - owner
+            - settings
+            - tags
+            - title
+        examples:
+          createCaseRequest:
+            $ref: '../components/examples/create_case_request.yaml'
+  responses:
+    '200':
+      description: Indicates a successful call.
+      content:
+         application/json; charset=utf-8:
+          schema:
+            type: object
+            properties:
+              $ref: '../components/schemas/case_response_properties.yaml'
+          examples:
+            createCaseResponse:
+              $ref: '../components/examples/create_case_response.yaml'
+  servers:
+     - url: https://localhost:5601
+
+delete:
+  description: >
+    Deletes one or more cases.
+    You must have all privileges for the **Cases** feature in the **Management**, **Observability**, or **Security** section of the Kibana feature privileges, depending on the owner of the cases you're deleting.
+  parameters:
+    - $ref: ../components/headers/kbn_xsrf.yaml
+    - name: ids
+      description: The cases that you want to removed. To retrieve case IDs, use the find cases API. All non-ASCII characters must be URL encoded.
+      in: query
+      required: true
+      schema:
+        type: string
+      example: '%5B%22d4e7abb0-b462-11ec-9a8d-698504725a43%22%5D'
```

Comment:
`%5B%22` at the beginning and `%22%5D` at the end are escaping characters and should be removed.

Reply:
Fixed in 0103fc39b083bcfe1cf43b5a596609fabde08529